src/server/hotkey_collector.h

```diff
@@ -63,10 +63,11 @@ class internal_collector_base;
 //    |                    |  |                     Hotkey                         |
 //    +--------------------+  +----------------------------------------------------+
 
-class hotkey_collector
+class hotkey_collector : public dsn::replication::replica_base
```

Comment:
Is it necessary to inherit `replica_base` class? Will hotkey_collector use other functions except log-printing functions like `ddebug_replica`?

Reply:
`hotkey_collector` will not use other functions related to `replica_base` without `ddebug_replica`